src/index.js

```diff
@@ -0,0 +1,60 @@
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import {
+  getLastPackageVersion,
+  getPackageVersion,
+  setPackageVersion,
+} from './local-settings'
+import initializeSentryIfEnabled from './sentry'
+
+// Initialize Sentry as early as possible, if enabled
+initializeSentryIfEnabled()
+
+const packageVersion = getPackageVersion()
+const lastPackageVersion = getLastPackageVersion()
+
+const [currentMajorVersion, currentMinorVersion] = packageVersion.split('.')
+const [lastMajorVersion, lastMinorVersion] = lastPackageVersion.split('.')
+
+// Purge localstorage when upgrading between different minor versions.
+if (
+  lastMajorVersion !== currentMajorVersion ||
+  lastMinorVersion !== currentMinorVersion
+) {
+  window.localStorage.clear()
+
+  // Attempt to clean up indexedDB storage as well.
+  if (
+    window.indexedDB &&
+    window.indexedDB.databases &&
+    window.indexedDB.deleteDatabase
+  ) {
+    // eslint-disable-next-line promise/catch-or-return
+    window.indexedDB
+      .databases()
+      .then(databases =>
+        databases.forEach(({ name }) => window.indexedDB.deleteDatabase(name))
+      )
+  }
+}
+
+// Save the current package version
+if (packageVersion !== lastPackageVersion) {
+  setPackageVersion(packageVersion)
+}
+
+// Enable hot module replacement
+if (module.hot) {
+  module.hot.dispose(function() {
+    // module is about to be replaced
+  })
+  module.hot.accept(function() {
+    // module or one of its dependencies was just updated
+  })
+}
+
+ReactDOM.render(<App />, document.getElementById('root'))
```

Comment:
Should we wrap the app in [`<React.StrictMode>`](https://reactjs.org/docs/strict-mode.html) since it is a new project?